src/core/ext/xds/xds_enabled_server.h

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright 2023 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#ifndef GRPC_SRC_CORE_EXT_XDS_XDS_ENABLED_SERVER_H
+#define GRPC_SRC_CORE_EXT_XDS_XDS_ENABLED_SERVER_H
+
+#include <grpc/support/port_platform.h>
```

Comment:
I don't think this include is actually needed.  I think our tools enforce it only if there is at least one include in the file, and in this case we don't need any.

Reply:
Removed